.github/workflows/actions/manage-doppler-secret/manage.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+# DOPPLER_TOKEN Must be defined in the ENV
+# The token is obtained from the doppler dashboard for your account
+DOPPLER_PROJECT=$1
+NAMESPACE=$2
+ENVIRONMENT=$3
+DOPPLER_OPERATOR_SECRET_NAME="${NAMESPACE}-${DOPPLER_PROJECT//_/-}"
+DOPPLER_CONFIG="${ENVIRONMENT}_${NAMESPACE}"
+
+function main(){
+  CURRENT_SECRET=$(get_current_secret)
+  if [ "$CURRENT_SECRET" = "SERVICE_TOKEN=" ]; 
+  then
+    echo "[create_doppler_secret] SERVICE_TOKEN is there but it's empty, deleting."
+    kubectl delete secret "$DOPPLER_OPERATOR_SECRET_NAME"
+    create_secret
+  elif [ "$CURRENT_SECRET" = "" ]; 
+  then
+    create_secret
+  else
+    echo "[create_doppler_secret] Secret is already in place."
+  fi
+}
+
+function get_current_secret() {
+  eval "kubectl get secret -n doppler-operator-system $DOPPLER_OPERATOR_SECRET_NAME --template=SERVICE_TOKEN={{.data.serviceToken}}"
```

Comment:
What is `.data.serviceToken` referring to here?

Reply:
It's a go template that's extractive the value of the SERVICE token from the secret.

Reply:
Oh duh, I see it now; it's part of the kubectl invocation.

Reply:
Saw your comment about this. I'll add a code comment for it.